MFVNeutralino/python/Vertexer_cff.py

```diff
@@ -16,11 +17,29 @@
 mfvVerticesAux = mfvSelectedVertices.clone(vertex_aux_src = 'mfvVerticesAuxPresel', min_ntracks = 3)
 
 mfvVertexSequenceBare = cms.Sequence(
+    mfvGenParticles * 
```

Comment:
Does this break anything for data, where we don't have a gen particle? Just curious...